tests/integration/test_hyper_transformer.py

```diff
@@ -1846,3 +1851,80 @@ def test_with_tuple_returned_by_faker(self):
             ]
         })
         pd.testing.assert_frame_equal(result, expected_results)
+
+    methods_to_inputs = {
+        'update_transformers': {'column_name_to_transformer': {'C': UniformEncoder()}},
+        'update_transformers_by_sdtype': {'sdtype': 'boolean', 'transformer': UniformEncoder()},
+        'remove_transformers': {'column_names': ['C']},
+        'remove_transformers_by_sdtype': {'sdtype': 'boolean'},
+    }
+
+    @pytest.mark.parametrize('method_name', methods_to_inputs.keys())
+    def test_unvalid_multi_column(self, method_name):
```

Comment:
minor: invalid

Reply:
Yes thanks, done in [e6083f7](https://github.com/sdv-dev/RDT/pull/758/commits/e6083f79907c1491785a1063acb9ae4804f15293)